spec/fixtures/application/1.0/means.json

```diff
@@ -18,9 +18,6 @@
 				}
 			]
 		},
-		"self_employment_details": {
-			"businesses": []
-		},
```

Comment:
I was pretty sure this wasn't where we landed with expressing this data in the json. I'll put it back if it should be there